docs/conf/portland/2018/cfp.rst

```diff
@@ -0,0 +1,105 @@
+
+Between the talks on stage, the discussions in the unconf, and the conversations in the hallway, 100% of the content for our conferences comes from our community! Whether your job title is writer, developer, product manager, support advocate, librarian, or one of a hundred others, your perspective on what makes good docs is what this conference is all about.
+
+Our goal is to give documentarians a chance to connect and learn from each other. You'll have the chance to compare notes on what's happening in the industry, dig into questions of convention and good practice, and generally nerd out about all the things we love about documentation.
+
+What we’re looking for
+----------------------
+
+The focus of Write the Docs is software documentation, but we actively seek
+talks that address a wide range of related subjects,
+at various levels of expertise. Documentation perspectives from other fields are welcome! Talks from the scientific community, fiction writers, system administrators, and support staff – in addition to technical writers and software programmers – are all valuable to our attendees.
+
+We welcome talks from first-time speakers, from industry experts, and from
+everyone in between, and across the board, we prefer hearing about new
+approaches than about tried-and-tested technology. We especially welcome talks
+from underrepresented groups within the tech community.
+
+We prefer talks backed by experience and experimentation to talks about theory,
+and we definitely don't like talks that bad-mouth technologies or approaches.
+Don't tell us why you hate something – tell us how you overcame the problems it was causing.
+
+We tend to avoid talks about specific tools, which often turn into marketing
+pitches or tutorials. We would much rather hear about process, culture, data,
+people, or the metaphysical side effects of spending your life thinking about docs.
+
+When crafting talk proposals remember that you're going to be talking to a mix
+of levels of expertise, skill sets, and professions. Your talk doesn't have to be
+relevant to everyone, but it should be relevant to most people and shouldn't
+make too many assumptions about what people already know.
+
+Take a look at the abstracts for accepted talks from the last couple of
+conferences for some ideas:
+
+* `Portland 2017 <http://www.writethedocs.org/conf/na/2017/speakers/>`_
+* `Portland 2016 <http://www.writethedocs.org/conf/na/2016/speakers/>`_
+* `Prague 2017 <http://www.writethedocs.org/conf/eu/2017/speakers/>`_
+* `Prague 2016 <http://www.writethedocs.org/conf/eu/2016/speakers/>`_
+
+Need help?
+-----------
+
+If you need a hand preparing your talk proposal, there are lots of good places to start:
+
+* **Community mentorship** – We have an ever-growing pool of previous Write the Docs speakers, many of whom are happy to be a second pair of eyes on talk proposals. If you're interested in working with a past speaker, let us know!
```

Comment:
We should probably direct them to slack instead of us, unless we have a specific plan here.

Reply:
Like, we should ask for mentors in advance, and publish them (good example here: https://www.pycascades.com/speakers/)